charting/control.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import annotations
+
+import string
+from dataclasses import dataclass, field
+from hashlib import md5
+from typing import Dict, Iterable, List
+from urllib.parse import urlencode
+
+import altair as alt
+import pandas as pd
+from cryptography.fernet import Fernet
+from django.conf import settings
+
+
+@dataclass
+class ChartBundle:
+    """
+    The chart bundle contains the dataframe and the altair chart.
+    To create multiple-rendering options.
+
+    """
+
+    label: str
+    alt_title: str
+    df: pd.DataFrame
```

Comment:
What's currently being returned by the chart is a styler. Both have similar `to_html' methods, so should be allowed as an option here.